{
    "original_source": {
        "database(url)": "https://www.cve.org",
        "software_system(CPE)": [
            "cpe:2.3:a:pimcore:customer_data_framework:*:*:*:*:*:*:*:*"
        ],
        "fix_commit": "https://github.com/pimcore/customer-data-framework/commit/72f45dd537a706954e7a71c99fbe318640e846a2",
        "version_end_excluding": {
            "pimcore:customer_data_framework": "3.4.2"
        },
        "path": {
            "file_path": ""
        }
    },
    "programming_language": "JSON",
    "licences": {
        "database": "",
        "affected_repository": "Other"
    },
    "related_vulnerability_entry": {
        "CVE": [
            "CVE-2023-4145"
        ],
        "CWE": [
            "CWE-79"
        ]
    }
}